tornado/test/simple_httpclient_test.py

```diff
@@ -238,6 +239,28 @@ def test_see_other_redirect(self):
             self.assertEqual("POST", response.request.method)
 
     @skipOnTravis
+    def test_connect_timeout(self):
+        # 0.1 and 0.5(nt) is copied from `self.test_request_timeout`
+        timeout = 0.1
+        timeout_min, timeout_max = 0.099, 0.15
+        if os.name == 'nt':
```

Comment:
I don't think we need this any more. Just use timeout_min = 0.09, timeout=0.1, and timeout_max=1.0. Since the `TimeoutResolver` never finishes we don't have to be very precise about the timing.